Update meteor-testapp to Meteor 1.8.2 / meteor-spk 0.5.0

kentonv opened the pull request:
Another update will be needed to work with Meteor 1.9, but first Sandstorm itself needs to be updated, which requires node-capnp to be updated for Node 12.

ocdtrekkie commented:
This PR appears to exchange the "babel runtime" error documented in #3225 with:

> Error: Cannot find module "capnp"

zenhack commented on meteor-testapp/package.json:
```diff
@@ -5,6 +5,7 @@
     "start": "meteor run"
   },
   "dependencies": {
+    "@babel/runtime": "^7.8.4",
```
Hm, should we have both `@babel/runtime` and `babel-runtime`? Not sure if there's actually a difference or if this is just a duplicate.

kentonv replied:
IIRC if I tried removing either one, it broke... but I have no idea, feel free to play with it further.

kentonv commented:
> This PR appears to exchange the "babel runtime" error documented in #3225 with:
>
> > Error: Cannot find module "capnp"

WTF it was totally working on my machine, but now I'm seeing that too.

kentonv commented:
I fixed `meteor-spk` to put the `capnp` module where Meteor will find it, and also updated it to Meteor 1.10.1, and updated the testapp to use this version.

xet7 commented:
@kentonv 

Have you uploaded that fixed meteor-spk? I get capnp errors when starting Wekan that uses meteor-spk 0.5.1.

kentonv commented:
@xet7 I just realized I hadn't pushed the git branch, did that now. But if you grabbed the 0.5.1 tarball, yes, that's the one that should work.
